frontend/libs/studio-pure-functions/src/BlobDownloader/BlobDownloader.test.ts

```diff
@@ -0,0 +1,28 @@
+import { BlobDownloader } from './BlobDownloader';
+
+describe('BlobDownloader', () => {
+  const data = { test: 'test' };
+  const blobDownloader = new BlobDownloader(JSON.stringify(data));
+  global.URL.createObjectURL = jest.fn();
+  global.URL.revokeObjectURL = jest.fn();
```

Comment:
n this case, relying on implementation details is actually ideal, since this class is the module that directly interacts with the URL API. 👏 🥳